### PR TITLE
added ability to search station list 

### PR DESCRIPTION
**Pull request by shaneoston72**
added ability to search to & from station lists, capture values selected and formatted list from uppercase to capitalize

### CODE REVIEW

File: www/lib/angular-sanitize/angular-sanitize.js [hunk 1]
```diff
@@ -33,36 +33,23 @@ var $sanitizeMinErr = angular.$$minErr('$sanitize');
...
- *   browser, won't make it through the sanitizer. The input may also contain SVG markup.
- *   The whitelist is configured using the functions `aHrefSanitizationWhitelist` and
- *   `imgSrcSanitizationWhitelist` of {@link ng.$compileProvider `$compileProvider`}.
+ *   it into the returned string.
+ *
+ *   The whitelist for URL sanitization of attribute values is configured using the functions
+ *   `aHrefSanitizationWhitelist` and `imgSrcSanitizationWhitelist` of {@link ng.$compileProvider
+ *   `$compileProvider`}.
+ *
+ *   The input may also contain SVG markup if this is enabled via {@link $sanitizeProvider}.
```
Comment (Managram) [inline www/lib/angular-sanitize/angular-sanitize.js:52]: maybe we should delete all of this commented text?

**Comment by rhiannonruth** [comment]
🚢

**Comment by MisaOgura** [comment]
:shipit:

**Comment by Managram** [comment]
:shipit:

**Pull request merged**